lib/maths/time_series/CTimeSeriesDecompositionDetail.cc

```diff
@@ -317,6 +317,8 @@ const core::TPersistenceTag LAST_CHANGE_POINT_TIME_7_11_TAG{"i", "last_change_po
 const core::TPersistenceTag LAST_CANDIDATE_CHANGE_POINT_TIME_7_11_TAG{
     "j", "last_candidate_change_point_time"};
 const core::TPersistenceTag LAST_CHANGE_POINT_7_11_TAG{"k", "last_change_point"};
+// Version 8.3
+const core::TPersistenceTag WINSORIZATION_DERATE_8_3_TAG{"l", "winsorization_derate"};
```

Comment:
It's probably best to update `MODEL_SNAPSHOT_MIN_VERSION` in `CAnomalyJob.cc` to `8.3.0` to match this.

Maybe it's not that critical if it's acceptable for the data behind this tag to get lost if a job moves around in a mixed version cluster.

But bumping the version that the Java side understands will help in the long run if we ever want to remove this.

Reply:
See [3adb43b](https://github.com/elastic/ml-cpp/pull/2280/commits/3adb43bb1eed624e6e2736c4f56b54e9c5302285).